backend/server.go

```diff
@@ -0,0 +1,183 @@
+package backend
+
+import (
+	rice "github.com/GeertJohan/go.rice"
+	"github.com/gomodule/redigo/redis"
+	"html/template"
+	"log"
+	"net/http"
+	"os"
+	"sort"
+	"strconv"
+	"strings"
+)
+
+type Data struct {
```

Comment:
nit: Do these structs need to be exported? (in Go, types and funcs starting with a capital letter are exported from the current package so they can be used elsewhere)

Reply:
You are right. That doesn't need to be capitalized. Thanks for the great nit(not really a nit)!!